scripts/Test-Solution.ps1

```diff
@@ -0,0 +1,52 @@
+#Requires -Version 6.0
+
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory=$false)]
+    [Alias('c')]
+    [ValidateSet('Debug', 'Release')]
+    [string]$Configuration = 'Debug',
+
+    [Parameter(Mandatory=$false)]
+    [Alias('or','open')]
+    [switch]$OpenReports
+)
+
+Set-StrictMode -Version 3.0
+$ErrorActionPreference = 'Stop'
+
+$RepositoryRootPath = (Get-Item -Path $PSScriptRoot).Parent
+$NuGetConfigurationFile = Join-Path -Path $RepositoryRootPath -ChildPath 'nuget.config'
+$SolutionFile = Join-Path -Path $RepositoryRootPath -ChildPath 'source' -AdditionalChildPath 'F0.Minesweeper.sln'
+$TestResultsDirectory = Join-Path -Path $RepositoryRootPath -ChildPath 'source' -AdditionalChildPath 'test', 'TestResults'
+$TestProjectFile = Join-Path -Path $RepositoryRootPath -ChildPath 'source' -AdditionalChildPath 'test', 'F0.Minesweeper.Logic.Tests', 'F0.Minesweeper.Logic.Tests.csproj'
+$CoverageReportTargetDirectory = Join-Path -Path $TestResultsDirectory -ChildPath 'coveragereport'
+$CoverageReportTargetFile = Join-Path -Path $CoverageReportTargetDirectory -ChildPath 'index.html'
+$StrykerConfigurationFile = Join-Path -Path $RepositoryRootPath -ChildPath 'source' -AdditionalChildPath 'test', 'stryker-config.json'
+
+dotnet clean $SolutionFile --configuration $Configuration --nologo --verbosity minimal
+
+dotnet restore $SolutionFile --configfile $NuGetConfigurationFile --verbosity minimal
+dotnet tool restore --configfile $NuGetConfigurationFile --verbosity minimal
+
+dotnet build $SolutionFile --configuration $Configuration --no-incremental --no-restore --nologo --verbosity minimal
+
+if (Test-Path -Path $TestResultsDirectory) {
+    Remove-Item -Path $TestResultsDirectory -Recurse
+}
+
+dotnet test $SolutionFile --configuration $Configuration --no-build --nologo --results-directory $TestResultsDirectory /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:CoverletOutput=./TestResults/
+
+dotnet tool run reportgenerator "-reports:$RepositoryRootPath/**/coverage.cobertura.xml" "-targetdir:$CoverageReportTargetDirectory" -reporttypes:HtmlInline_AzurePipelines
+
+$TestProjectFile = $($TestProjectFile.Replace('\', '/'))
+dotnet tool run dotnet-stryker --solution-path $SolutionFile --test-projects "['$TestProjectFile']" --config-file-path $StrykerConfigurationFile
```

Comment:
I just tried to run this script in my other branch.
As a FYI: I downloaded the following files:
* scripts/Test-Solution.ps1
* .config/dotnet-tools.json
* nuget.config

First of all, I had to update Powershell/install a new Powershell version, as per default I only had Powershell 5.1.
The script required at least Powershell 6.0, and with the "official" Script `iex "& { $(irm https://aka.ms/install-powershell.ps1) } -UseMSI"`
This installed Powershell 7 💪 

And then, Stryker didn't run for me with many errors like this:
`[22:33:48 WRN] Stryker.NET encountered an compile error in C:\Users\[...]\F0.Minesweeper.Logic\Cell.cs (at 6:34) with message: The feature 'init-only setters' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version. (Source code: init;)`

As a solution, I added `--language-version preview` to the end of this command to specify a newer C# version.
Do we want to generally add this option/switch to this Stryker call? Let's discuss next time 😄

Reply:
Cool, adding the `--language-version preview`